_about/employment-opportunities.html

```diff
@@ -0,0 +1,29 @@
+---
+title: Employment Opportunities
+---
+
+<p class="mt-3">
+    The OSG Consortium is proud to display the following positions dedicated to advancing the state of research computing.
+</p>
+
+{% include /get/jobs.liquid %}
+
+{% assign internal_jobs = jobs | where_exp: "x", "x.type == 'internal'" %}
+{% if internal_jobs.size > 0 %}
+    {% for job in internal_jobs %}
+        {% include /cards/job.html %}
+    {% endfor %}
+{% endif %}
+
+{% assign external_jobs = jobs | where_exp: "x", "x.type == 'external'" %}
+{% if external_jobs.size > 0 %}
+    <h2 class="mt-4" id="external-positions">OSG Affiliate Institution Positions</h2>
```

Comment:
Need to work on the text of this heading… maybe something like “Other Positions of Possible Interest”. We really want to make clear that we do not endorse or sponsor these positions in any way.